v22.1/show-grants.md

```diff
@@ -53,6 +53,7 @@ Field            | Description
 `type_name`      | The name of the user-defined type.
 `grantee`        | The name of the user or role that was granted the [privilege](security-reference/authorization.html#managing-privileges).
 `privilege_type` | The name of the privilege.
+`is_grantable`   | <span class="version-tag">New in v22.1:</span> `TRUE` if the grantee has the admin option on the table; `FALSE` if not.
```

Comment:
it should say "has the grant option on the privilege"

(that's an important distinction -- grant option is a flag that says what a user can do with a GRANT they already have)

Reply:
ah, i forgot one point: the word "table" is too specific here, since SHOW GRANTS will also show grants on databases, schemas, and types.

Reply:
thanks!

Reply:
Sorry, missed your second comment before I pushed the edit. Is "grant option on the privilege" correct?

Reply:
no problem! i think "grant option on the object" or "grant option for the privilege" are the most correct ways of phrasing this.

Reply:
should the database/type/schema examples be updated as well?